ctms/bin/acoustic.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+"""Schedule contacts to be synced to Acoustic."""
+import os
+import sys
+from typing import Optional, TextIO
+
+import click
+
+from ctms import config
+from ctms.crud import (
+    bulk_schedule_acoustic_records,
+    get_contacts_from_newsletter,
+    get_contacts_from_waitlist,
+)
+from ctms.database import SessionLocal
+from ctms.exception_capture import init_sentry
+from ctms.log import configure_logging
+
+
+@click.group()
+@click.pass_context
+def cli(ctx):
+    ctx.obj["dbsession"] = SessionLocal()
```

Comment:
I think `ctx.obj["dbsession"] = SessionLocal()` will open a session that we never close, since we manually close it in the [`get_db`](https://github.com/mozilla-it/ctms-api/blob/998124cb794850a8017aecbb03a95f85fa42f35e/ctms/app.py#L124-L129) dependency.

What we could do is `ctx.obj["SessionLocal"] = SessionLocal`

Then below:
```python
with ctx.obj["SessionLocal"]() as dbsession:
    return do_resync(dbsession, emails_file, newsletter, waitlist)
```

Or, since SessionLocal is a factory anyway, I think we could just use the global directly:
```python
with SessionLocal() as dbsession:
    return do_resync(dbsession, emails_file, newsletter, waitlist)
```

Reply:
I realized that and then thought "ah it's a short-lived script anyway" 

But you're right, it's cleaner